src/main/kotlin/org/rust/ide/annotator/RsErrorAnnotator.kt

```diff
@@ -354,6 +355,11 @@ class RsErrorAnnotator : AnnotatorBase(), HighlightRangeExtension {
         binding.ancestorStrict<RsValueParameterList>()?.let { checkDuplicates(holder, binding, it, recursively = true) }
     }
 
+    private fun checkPatIdent(holder: RsAnnotationHolder, patIdent: RsPatIdent) {
+        if (patIdent.pat !is RsPatRest) return
```

Comment:
Whats' about patterns without bindings?
```rust
fn main() {
    let [x, ..] = [1, 2, 3];
}
```

Reply:
Don't forget to check that you don't have false positive for tuple patterns `let (x, .., y)  = (1, 2, 3);` and all other places where we use `RsPatRest`